### ISSUE TITLE
Showing zero duration for past or future

### ISSUE THREAD

**Issue by ypresto** [ocpsoft/prettytime#111]
We are using prettytime library to show when a comment is wrote by a user.
When user adds comment, the server sometimes replies with the date slightly in the future than client date.
So we normalizes the date like below:

``` java
if (date.after(referenceDate)) date = referenceDate;
```

But it shows "moments from now" (future), instead of "moments ago" (past).
This library seems to treat zero duration as future.

How about adding methods to format only past (`formatPast()`) or future (`formatFuture()`) date?
Also it might be better if there are tests about zero duration.
(Currently `testRightNow` uses two `new Date()` instance so it has duration.)

Thanks!

**Comment by lincolnthree** [comment]
While this would seem to be simple. (And apologies for not responding for 5 years.) If we wanted to make this change (and I'm not sure that we do,) this would involve a major overhaul to the library's test suite, that we don't really have the capacity to perform without contacting each translator to update the tests for their bundle.

That said, this should be simple to resolve. Simply add/subtract enough milli/nanosecond to the reference date to ensure that it's actually providing a past value.

`testRightNow` indeed intentionally uses two `new Date()` objects to verify  this behavior.

In short, this is working as intended.
